package.json

```diff
@@ -9,10 +9,15 @@
   "author": "Mark Erikson <mark@isquaredsoftware.com>",
   "license": "MIT",
   "devDependencies": {
-    "babel-core": "^6.26.0",
-    "babel-eslint": "^7.2.3",
-    "babel-plugin-external-helpers": "^6.22.0",
-    "babel-preset-env": "^1.6.1",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-external-helpers": "^7.0.0",
```

Comment:
Thanks for the hint - I'll look into that.

Reply:
I checked again and `@babel/plugin-external-helpers` is actually not used. I installed it because it was in the Babel 6 setup, but realized it shouldn't be there and forgot to remove the package. Will remove.